src/service-worker/core.js

```diff
@@ -150,41 +119,19 @@ function fetchAndCache(request) {
       // This intentionally does not block the request resolution to speed
       // things up. This is likely fine since you don't have multiple
       // `<script>`s with the same `src` on a page.
-      db.transaction('js-files', 'readwrite').run(transaction => {
-        const files = transaction.objectStore('js-files');
-        // Push the item into our cached files. We do this first so it might be
-        // caught if the user refreshes quickly.
-        return files.add({
-          file: requestFile,
-          version: requestVersion,
-        }).then(key => {
-          // Now, let's prune all the older versions.
-          return files.index('files').openCursor(requestFile).while(cursor => {
-            // Don't prune the file-version we just added.
-            if (cursor.primaryKey === key) {
-              return '';
-            }
-
-            // We'll want to remove this from the cache, too.
-            const version = cursor.value.version;
-            cursor.delete();
-            return version;
-          });
-        });
-      }).then(removedVersions => {
-        removedVersions.forEach(version => {
-          // Is the return value for the file-version we added (we explicitly
-          // returned an empty string for it)?
-          if (!version) {
-            return;
+      cache.keys().then(requests => {
```

Comment:
Update the block comment (line above stills says // Store the file version in IndexedDB0)